src/Discord.Net.Core/Entities/Guilds/IGuildIntegration.cs

```diff
@@ -1,17 +1,34 @@
-﻿using System;
+using System;
 
 namespace Discord
 {
     public interface IGuildIntegration
     {
+        /// <summary> Gets the integration ID. </summary>
+        /// <returns> An <see cref="ulong"/> representing the unique identifier value of this integration. </returns>
         ulong Id { get; }
+        /// <summary> Gets the integration name. </summary>
+        /// <returns> A string containing the name of this integration. </returns>
         string Name { get; }
+        /// <summary> Gets the integration type (twitch, youtube, etc). </summary>
+        /// <returns> A string containing the name of the type of integration. </returns>
         string Type { get; }
+        /// <summary> Gets if this integration is enabled or not. </summary>
+        /// <summary> A value indicating if this integration is enabled. </returns>
         bool IsEnabled { get; }
+        /// <summary> Gets if this integration is syncing or not. </summary>
+        /// <returns> A value indicating if this integration is syncing. </returns>
```

Comment:
If possible, perhaps add a `<remarks>` explaining what syncing means?

Reply:
This I wasn't too sure about (I'm not a partnered Twitch or Youtube streamer), but I was able to ask in the `#api` channel for some extra info.